README.md

```diff
@@ -36,3 +36,17 @@ Then use the following command to install the python library.
 
 This repository contains lua bindings for libcircllhist, which are generated by `make`,
 and will be installed by `make install` to the directory `/usr/local/share/lua/5.1/`.
+
+## FAQ
+
+### After the installation, libcircllhist.so can not be found by my program / the python bindings. How can I fix this?
```

Comment:
If this has been a problem for users, we could also embed the native library inside the Python module..

Reply:
Yeah. I was not sure how to do that. We would need different libs for x64/x32 and potentially multiple OSes. I did not want to go into this. Maybe this a problem for the C layer to address. We do have EL7/Ubuntu packages AFAIK.